### ISSUE TITLE
AppImage:  libffi & libtiff error

### ISSUE THREAD

**Issue by janakdom** [ente-io/ente#472]
After i download pre-release of Nightly Build AppImage and run it i got error:

./ente_auth: error while loading shared libraries: libffi.so.7: cannot open shared object file: No such file or directory

after that i install latest libffi7 i got:

./ente_auth: symbol lookup error: /usr/lib/libtiff.so.6: undefined symbol: jpeg12_write_raw_data, version LIBJPEG_8.0

and i hane no idea how to fix it.

I have latest Manjaro Linux with KDE & Wayland.

![image](https://github.com/ente-io/auth/assets/10292894/00d4bdc0-d19e-4eae-9b12-849081c446d0)

**Comment by mrbluecoat** [comment]
Similar on Ubuntu 22.04 x86_64. I had to run `sudo apt install libffi7` to get pas the ffi error and now I get:

```
$ LIBGL_DEBUG=verbose ./auth-linux-v2.0.34-beta0-x86_64.AppImage
(ente_auth:17604): Gdk-CRITICAL **: 18:57:35.241: gdk_window_get_state: assertion 'GDK_IS_WINDOW (window)' failed
libGL: Can't open configuration file /etc/drirc: No such file or directory.
libGL: Can't open configuration file /home/mbc/.drirc: No such file or directory.
libGL: using driver i915 for 11
libGL: pci id for fd 11: 8086:1912, driver iris
libGL: MESA-LOADER: failed to open /usr/lib/x86_64-linux-gnu/dri/iris_dri.so: usr/lib/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /usr/lib/x86_64-linux-gnu/dri/iris_dri.so)
...
libGL error: failed to load driver: iris
libGL: MESA-LOADER: failed to open /usr/lib/x86_64-linux-gnu/dri/swrast_dri.so: usr/lib/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /usr/lib/x86_64-linux-gnu/dri/swrast_dri.so)
```

**Comment by mrbluecoat** [comment]
FYI, this only occurred with the AppImage. The `deb` option worked great.

**Comment by dtantono** [comment]
Yes, I got the stable version 2.0.5 for ubuntu 22.04 and I have same issue

**Comment by ebigaignon** [comment]
Hello,
The error is still present on Debian 12, with v2.0.54: ./ente_auth: error while loading shared libraries: libffi.so.7

**Comment by wehagy** [comment]
The issue persist on archlinux with ente-auth-v2.0.54-x86_64.AppImage
```sh
~$ ./ente-auth-v2.0.54-x86_64.AppImage 
./ente_auth: error while loading shared libraries: libffi.so.7: cannot open shared object file: No such file or directory
```

**Comment by mrbluecoat** [comment]
Thanks for the heads up, @wehagy - `ente-auth-v3.0.7-x86_64.AppImage` worked great for me

**Comment by janakdom** [comment]
ente-auth-v3.0.8-x86_64.AppImage still not working for me. Same error.
```
./ente-auth-v3.0.8-x86_64.AppImage
./ente_auth: symbol lookup error: /usr/lib/libtiff.so.6: undefined symbol: jpeg12_write_raw_data, version LIBJPEG_8.0
```

**Comment by janakdom** [comment]
@dtantono i use latest full Manjaro with KDE 6

**Comment by 103sbavert** [comment]
it used to work just fine a few weeks ago and now it doesn't work at all.

**Comment by prateekmedia** [comment]
@103sbavert Did you updated your system?

**Comment by 103sbavert** [comment]
Yes, I'm running the latest version of all packages on Arch Linux @prateekmedia

**Comment by 103sbavert** [comment]
Any updates on this?

**Comment by neeraj-pilot** [comment]
This should be fixed now. Please retest with the latest Auth AppImage [v4.4.23](https://github.com/ente-io/ente/releases/tag/auth-v4.4.23).

The current AppImage packaging changed how runtime libraries are loaded. If it still fails, please share:
- exact AppImage filename/version
- distro and desktop session, e.g. X11 or Wayland
- terminal output from running the AppImage
- whether `./ente-auth-...AppImage --appimage-extract && ./squashfs-root/enteauth` works
